frontend/assessments/2/script.js

```diff
@@ -0,0 +1,38 @@
+const messageInput = document.getElementById("inp");
+const sellButton = document.getElementById("sell");
+const buyButton = document.getElementById("buy");
+const stock = document.getElementById("stock");
+const price = document.getElementById('pnum');
```

Comment:
That's what I mentioned above these will throw error
SInce your DOM isn't loaded yet